npm-debug.log

```diff
@@ -0,0 +1,26 @@
+0 info it worked if it ends with ok
```

Comment:
can you remove this file, and add an entry to npm ignore file so it wont be included again